inc/vcf/string_constants.hpp

```diff
@@ -174,6 +174,14 @@ namespace ebi
     const std::string PQ = "PQ";
     const std::string PS = "PS";
 
+    //Extensions of compressed files
```

Comment:
Add a space after the comment begins `//`
I would prefer this line as `// Compressed file extensions`